.nf-core.yml

```diff
@@ -4,6 +4,7 @@ lint:
     - params.input
   files_unchanged:
     - .gitignore
+    - docs/images/nf-core-mcmicro_logo_dark.png
```

Comment:
See my recent comment on #66 , I think we can restore the old png instead of adding this to the ignore list.

Reply:
Great, thanks.